### ISSUE TITLE
Enable the enduser to download the kubeconfig

### ISSUE THREAD

**Issue by mak3r** [mak3r/turnkey#13]
After the user sets up their local network, they should be able to download the kubeconfig so they can access the cluster from another system.

**Comment by mak3r** [comment]
currently the last page of the UI flow gives the user information on how to connect to the device and get the kubeconfig. Moving this out of MVP and marking as an [RFE]

**Comment by bgulla** [comment]
Crazy idea: it could be possible to generate the token and CA **pre** k3s-bootstrap and present the user with the generated kubeconfig file before the node bootstraps (**I think**). This would negate the need to necessarily ssh into the box just to download the kubectl file.

**Comment by mak3r** [comment]
Yes it is possible to create the token, ca and certs - essentially building a kubeconfig from scratch and forcing those as the underlying certs for the cluster. I'm not sure how cert rotation would be done in this scenario. And of course the process for building the cluster would need to be modified to accommodate this approach.
